.github/ISSUE_TEMPLATE/new-plugin.md

```diff
@@ -0,0 +1,24 @@
+---
+name: New plugin
+about: Open Fixture Library should have a plugin for this fixture format.
+title: Add [software / console name] Plugin
+labels: new-plugin
+assignees: ''
+
+---
+
+XXXXX Please edit all lines that start with "XXXXX".  
+XXXXX Short description of the software / desk console using the fixture format.
+
+- **Link to the software / desk console website:**  
+  XXXXX
+- **Where can fixtures of this format be downloaded?**  
+  XXXXX
+- **Is the format binary?**  
+  XXXXX Yes / No / Unknown
```

Comment:
Rather something like:

What is the fixture format? binary / XML / JSON / unknown